xil_carma_cloud/cdasim_config/carma/inlanecruising_config/parameters.yaml

```diff
@@ -1,9 +1,9 @@
 # The length of the trajectory in time domain, in seconds
 trajectory_time_length: 6.0 # Trajectory length in seconds
 curve_resample_step_size: 1.0 # Curve re-sampling step size in m
-default_downsample_ratio: 18 # Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
-turn_downsample_ratio: 20 # Amount to downsample input lanelet centerline data on turns. Value corresponds to saving each nth point.
-minimum_speed: 2.2352 # Minimum allowable speed in m/s
+default_downsample_ratio: 4 # Amount to downsample input lanelet centerline data. Value corresponds to saving each nth point.
+turn_downsample_ratio: 4 # Amount to downsample input lanelet centerline data on turns. Value corresponds to saving each nth point.
```

Comment:
these are needed otherwise platform will not be able to fit spline due to 2mph. This is too short distance to be downsampling 18-20 points.